packages/components/src/atoms/PaneCloseIcon/PaneCloseIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import * as S from "./PaneCloseIcon.styled";
+
+export type PaneCloseIconType = {
+  onClick: () => void;
+  containerId?: string;
+  containerStyle?: React.CSSProperties;
+  iconStyle?: React.CSSProperties;
+};
+
+const PaneCloseIcon: React.FC<PaneCloseIconType> = (props) => {
```

Comment:
oh no I have to guess we need to have this battle soon :D 

I'm team

```
type Props = {
 containerId: string;
  // etc..
}

export function PlaneCloseIcon({
  containerId,
  container
  iconStyle
  onClick
} : Props) {
  return ..
}
```

It's just more natural way of programming since that's how you'd write any function.

Reply:
I guess it's more a matter of preference here, I'm just used to write arrow functions when creating components 😄